.ci/cico_do_docker_build_tag_push.sh

```diff
@@ -12,7 +12,7 @@ ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 RH_CHE_TAG=$(git rev-parse --short HEAD)
 
-UPSTREAM_TAG=$(sed -n 's/^revision = \(.\{7\}\).*/\1/p' ${ABSOLUTE_PATH}/../assembly/assembly-build-info/target/dependency/WEB-INF/classes/org/eclipse/che/ide/ext/help/client/BuildInfo.properties)
```

Comment:
Afaik the information in the build-info was expected to be the one of *upstream*.
It doesn't seem to be the case anymore.

The idea was that the `rh-che` docker image tag would allow knowing the upstream Che commit on which it is based.

Reply:
I thought that the main goal is to have different tags for different builds to be able to deploy any of them or roll back in case of an unsuccessful update.

So BuildInfo is not packaged into assembly-ide-war. If it is required to know the upstream Che commit I can create an issue in eclipse/che.

Reply:
@mario wdyt ?

Reply:
Having the upstream commit is kinda of useful. When running a che server image we can easily associate it to the right upstream and downstream commit. And we can deduce if an upstream fix is included in the image or not. That said that is not a blocker but a nice to have. So yes please @sleshchenko open and issue on upstream Che to discuss that.

Reply:
Ok, I'll create an issue for that

Reply:
FYI I've discovered that each jar contains `META-INF/MANIFEST.MF` that has the line with revision number like `SCM-Revision: b956dfbacdf17b56c0b5d6b111d6858a9f2dba02`. 
Anyway, I'll create an issue, describe it there and then it will be decided which way is the best to do that.

Reply:
OK, thanks

Reply:
I've create an issue for that https://github.com/eclipse/che/issues/7854

Reply:
@davidfestal @l0rd I've fixed usage of eclipse revision number in the following commit https://github.com/redhat-developer/rh-che/pull/461/commits/ff1385c2ca9bf8e42bfe17ee1608b7cd5e2c1c47